### ISSUE TITLE
Atom editor not working

### ISSUE THREAD

**Issue by MikeWortin** [voidlinux/void-packages#14263]
### System

* xuname:  
  *Void 4.16.7_1 x86_64 GenuineIntel uptodate rrFF*
* package:  
  *atom-1.26.1_1*

### Expected behavior
Editor opening and working normally as expected.

### Actual behavior
Nothing shows on the screen, if I run ``killall atom`` atom shows message, that it was crashed

### Steps to reproduce the behavior
Run ``atom`` command or open Atom from application menu.

**Comment by noblehelm** [comment]
Can confirm. I did install `atom-1.26.1_1` and it did not work on my machine. Tried running from terminal, not output whatsoever, and options like `--dev`, `--foreground`, `--safe` and even `--log-file` do not work. Maybe something to do with the recent `fontconfig` update that broke browsers, as stated in #14161.

**Comment by noblehelm** [comment]
Well, might be related, since atom uses Electron (which in turn uses Chromium/Chrome tech), and the fontconfig issue does affect Opera and Chrome. Running `atom -v` gives:
```
Atom    : 1.26.1
Electron: 1.7.11
Chrome  : 58.0.3029.110
Node    : 7.9.0
```
As for it working with the official one from the website, maybe that one is built statically (static libs) and the void pkg isn't? I don't really know, to be honest. I just confirm that I have the same issue.

**Comment by wpbirney** [comment]
atom is up to 1.27.0 now, maybe updating our package to that will help

**Comment by solbjorn** [comment]
atom-1.28.0-beta3 built with xbps-src is fully workable, so I think we might wait for stable 1.28 release and then update a template (remind that Electron 2 has been switched from GTK2 to GTK3, so it will be needed to change `gtk+` makedep to `gtk+3`).

**Comment by wpbirney** [comment]
@solbjorn agreed, can confirm 1.28.0-beta3 builds and runs here fine. It is odd that the downloaded version runs fine. AFAIK they are packaged using the same method, the downloaded one probably comes with the extra deps to be more portable across distros

**Comment by solbjorn** [comment]
Yep, but they may build it on more outdated environment like Debian stable or smth like that.
